blocks/subscriptions-block/components/usePaymentOptions.jsx

```diff
@@ -1,65 +1,79 @@
 import { useState, useEffect } from "react";
 
-import useSales from "./useSales";
+import { useSales } from "@wpmedia/arc-themes-components";
 
 const STRIPE_PAYMENT_METHOD_ID = 18;
 const PAYPAL_METHOD_ID = 10;
 
-const getPaymentMethodByID = (paymentOptions, paymentMethodTypeID, paymentMethodID) => {
+export const getPaymentMethodByID = (paymentOptions, paymentMethodTypeID, paymentMethodID) => {
+	let minStripeIntentsID;
+
 	const stripeDefault = paymentOptions?.find(
 		(opts) =>
 			opts?.paymentMethodType === paymentMethodTypeID && opts?.paymentMethodID === paymentMethodID,
 	);
 
 	if (stripeDefault) {
 		return stripeDefault;
-	} else {
+	}
+
+	if (!stripeDefault) {
```

Comment:
If the first condition is truthy and returns the second will never be hit.  If it's falsey, then it will be falsey again on the second `if`.

(the second `if` is redundant)